pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -59,14 +59,14 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, pr *v1beta1.PipelineRun)
 func (r *Reconciler) FinalizeKind(ctx context.Context, pr *v1beta1.PipelineRun) pkgreconciler.Event {
 	// Check to make sure the PipelineRun is finished.
 	if !pr.IsDone() {
-		logging.FromContext(ctx).Infof("pipelinerun is still running")
+		logging.FromContext(ctx).Debugf("pipelinerun \"%s/%s\" is still running", pr.Namespace, pr.Name)
```

Comment:
Instead of plumbing through the namespace/name in each log statement, you can set this in the context:

```go
log := logging.FromContext(ctx).With(
  "pipelinerun", fmt.Sprintf("%s/%s", pr.Namespace, pr.Name),
)
ctx = logging.WithLogger(ctx, log)
```

This will make sure that other logs further down the callstack have similar fields set!